embedded-graphics/src/primitives/thick_segment.rs

```diff
@@ -0,0 +1,105 @@
+//! A line segment constructed from two line joints.
+//!
+//! Segment always has one open side, so scanline intersections may return one or two points, but
+//! no more.
```

Comment:
IMO this comment belongs to the `ThickSegment` struct and not the module.

Reply:
This comment isn't even correct anymore - the thick segment is now always generated as a closed shape, guaranteeing a line intersection.